packages/docs/src/pages/index.js

```diff
@@ -9,55 +9,292 @@ import { Hero } from '@site/src/components/Hero'
+const ExampleImage2 = require('@site/static/img/example2.png').default
+
+const Button = ({ style, children, href }) => {
+    return (
+        <a href={href} target="_blank">
+            <div
+                style={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                }}
+            >
+                <div
+                    style={{
+                        textAlign: 'center',
+                        cursor: 'pointer',
+                        backgroundColor: '#A3ECE1',
+                        borderRadius: '32px',
+                        height: '64px',
+                        fontSize: '20px',
+                        fontWeight: 600,
+                        lineHeight: '28px',
+                        color: 'black',
+                        userSelect: 'none',
+                        padding: '18px 40px',
+                        ...(style ?? {}),
+                    }}
+                >
+                    {children}
+                </div>
+            </div>
+        </a>
+    )
+}
+
+const Section = ({ title, description }) => (
+    <div
+        style={{
+            display: 'flex',
+            flexDirection: 'column',
+            borderTop: '1px solid black',
+            width: '250px',
+            marginTop: '56px',
+            color: 'black',
+        }}
+    >
+        <div style={{ height: '24px' }} />
+        <div style={{ fontSize: '20px', fontWeight: 700, lineHeight: '28px' }}>
+            {title}
+        </div>
+        <div style={{ height: '24px' }} />
+        <div className={styles.paragraph}>{description}</div>
+    </div>
+)
 
 function HomepageHeader() {
     const { siteConfig } = useDocusaurusContext()
     return (
-        <header className={clsx('hero hero--primary', styles.heroBanner)}>
+        <div
+            style={{
+                maxWidth: '1196px',
+                margin: 'auto',
+                paddingLeft: '16px',
+                paddingRight: '16px',
+                paddingTop: '104px',
+                display: 'flex',
+                flexDirection: 'column',
+            }}
+        >
+            <div
+                style={{
+                    fontSize: '96px',
+                    fontWeight: 700,
+                    lineHeight: '100%',
+                    alignSelf: 'center',
+                }}
+            >
+                A protocol built for handling user data anonymously.
+            </div>
+            <div style={{ height: '24px' }} />
+            <div className={styles.paragraph} style={{ maxWidth: '768px' }}>
+                UniRep is a Zero-Knowledge Protocol for user data & reputation
+                management. We use pioneering technology to offer a space for
+                developers and users alike to explore the potential of
+                privacy-centered online interactions.
```

Comment:
@ChialiT I think we should talk about how UniRep can do 
e.g. Users of UniRep can receive data anonymously and prove data with the user's consent.

in this paragraph I think it is hard to understand what it is different from other zk applications and why we use this protocol.